config/config.go

```diff
@@ -0,0 +1,49 @@
+package config
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	logger "github.com/canonical/edgex-snap-info/log"
+)
+
+type Config struct {
+	Snaps map[string]struct {
+		GithubRepo string
+	}
+}
+
+func LoadConfig(confFile string) (c *Config, err error) {
```

Comment:
```suggestion
func Load(confFile string) (c *Config, err error) {
```
so the caller would call `config.Load`.

In addition: `confFile` -> `path`